src/NHibernate.Test.VisualBasic/NHibernate.Test.VisualBasic.vbproj

```diff
@@ -29,7 +29,9 @@
     <None Include="..\NHibernate.Test\App.config" Link="App.config" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
     <PackageReference Include="Microsoft.VisualBasic" Version="10.2.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.10.0" />
```

Comment:
Not having directly these references in the VB project was not allowing Rider to run the unit tests.